src/LiveProofs.lhs

```diff
@@ -1038,6 +1039,21 @@ dispLiveProof maxm liveProof
        ) )
  where (trm,sc) = unwrapASN $ conjecture liveProof
 
+
+-- dispLiveProof but when proof is complete
+-- temporary
+dispEndProof :: LiveProof -> String
+dispEndProof liveProof
+ = unlines
+     ( ( ("\nProof for "++red (widthHack 2 $ conjName liveProof))
+       : ("\t" ++ green(trTerm 0 trm ++ "  "++ trSideCond sc))
+       : ("by "++strategy liveProof)
+       : map shLiveStep (reverse (stepsSoFar liveProof))
+       ))
```

Comment:
this is nice - worth keeping!  I see that is based on lines 1027-30 in this file.

As we plan to use PRs for workflow, I am going to suggest a change before accepting this one.

Can you define a function at line 1040 (just before `shLiveStep`) that does the above four lines, and replace 1027-30 and new 1048-512 with a call to that function?